validator/client/validator.go

```diff
@@ -387,6 +386,12 @@ func (v *validator) checkAndLogValidatorStatus(statuses []*validatorStatus, acti
 			}
 		case ethpb.ValidatorStatus_ACTIVE, ethpb.ValidatorStatus_EXITING:
 			validatorActivated = true
+			if status.status.Status == ethpb.ValidatorStatus_ACTIVE {
```

Comment:
Maybe separate the two cases `ethpb.ValidatorStatus_ACTIVE` and `ethpb.ValidatorStatus_EXITING` to avoid the
```go
if status.status.Status == ethpb.ValidatorStatus_ACTIVE
```

in the switch/case?